specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/defenderSettings.json

```diff
@@ -0,0 +1,566 @@
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create or update IoT Defender settings": {
+            "$ref": "./examples/DefenderSettings/Put.json"
+          }
+        },
+        "tags": [
+          "IoT Defender Settings"
+        ],
+        "operationId": "DefenderSettings_CreateOrUpdate",
+        "description": "Create or update IoT Defender settings",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/DefenderSettingsModel"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated IoT Defender settings",
+            "schema": {
+              "$ref": "#/definitions/DefenderSettingsModel"
+            }
+          },
+          "201": {
+            "description": "Created IoT Defender settings",
+            "schema": {
+              "$ref": "#/definitions/DefenderSettingsModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete IoT Defender settings": {
+            "$ref": "./examples/DefenderSettings/Delete.json"
+          }
+        },
+        "tags": [
+          "IoT Defender Settings"
+        ],
+        "operationId": "DefenderSettings_Delete",
+        "description": "Delete IoT Defender settings",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "IoT Defender settings deleted"
+          },
+          "204": {
+            "description": "IoT Defender settings does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTSecurity/defenderSettings/default/packageDownloads": {
```

Comment:
POST actions should typically be a verb.  I.e. listPackageDownloads.  The list prefix will also let users call this from a template deployment

Reply:
Please notice this API is a copy-paste from an existing API we have under _Microsoft.Security_